DOCS-1375: Update I2C config attributes and screenshot for adx1345 accelerometer (#2191)
viamrobotics/docs: docs/components/movement-sensor/adxl345.md

```json {class="line-numbers linkable-line-numbers"}
{

>   "board": "<your-board-name>",

Reviewer: I'd order these the same as the attribute descriptions below. If you like my other suggestion, move this line down below "use_alternate_i2c_address" and above "tap". That way, the required attributes (i.e., only "i2c_bus" any more) are at the top.

Author: Done

      "namespace": "rdk",
      "attributes": {

>         "board": "local",

Reviewer: We can remove this line, and frankly we can remove the entire board component above (lines 110-123). This example doesn't use any interrupt pins, so we no longer depend on the board at all.

Author: Sounds good!

>         "i2c_bus": "1",

Reviewer: This says "1", but line 60 gave the example "2" and the new screenshot uses "2". Should they be different to show that you can have different values, or the same to be consistent? Maybe it's right the way you've written it; I'm just looking for weird stuff.

Author: I'm good with changing it to 1, just didn't catch that difference!

>         "i2c_bus": "1",

Author: ```suggestion
        "i2c_bus": "2",
```

## Attributes

<!-- prettier-ignore -->
| Name | Type   | Inclusion    | Description |
| ---- | ------ | ------------ | ----------- |

> | `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |

Reviewer: The point of the RDK change is that you don't configure the I2C bus on the board component any more, and this component is no longer dependent on the board at all. I like the phrasing in https://docs.viam.com/components/board/pca9685/: "The index of the I2C bus on the board your device is connected to. Often a number. Example: `"1"`"

Author: Oh yes, thank you for catching this! Updating

> | `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |

Author: ```suggestion
| `i2c_bus` | string | **Required** | The index of the I2C bus on the board your device is connected to. Often a number. <br> Example: "1"  |
```

> | `board` | string | Optional | The `name` of the [board](/components/board/) to which the device is wired. Only needed if you've configured any [interrupt](/components/board/#digital_interrupts) functionality. |

Author: ```suggestion
| `use_alternate_i2c_address` | bool | Optional | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
| `board` | string | Optional | The `name` of the [board](/components/board/) to which the device is wired. Only needed if you've configured any [interrupt](/components/board/#digital_interrupts) functionality. |
```

> | `use_alternate_i2c_address` | bool | Optional | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |

Reviewer: I'm open to disagreement, but I'd swap lines 148 and 149. That way, the board is right next to the tap and free_fall sections that interact with it, and the fields are roughly ordered by popularity (use_alternative_i2c_address is probably used more often than board will be after this change is out).

Author: Done!

> | `use_alternate_i2c_address` | bool | Optional | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |

Author: ```suggestion
```
